app/controllers/users/piv_cac_authentication_setup_controller.rb

```diff
@@ -98,6 +98,7 @@ def process_valid_submission
       )
       create_user_event(:piv_cac_enabled)
       Funnel::Registration::AddMfa.call(current_user.id, 'piv_cac')
+      session[:needs_to_setup_piv_cac_after_sign_in] = false
```

Comment:
Before, this wasn't being set while processing a valid PIV submission, causing the `after_sign_in_path_for(current_user)` to put the user back to the PIV setup screen, even though they just set it up.

Reply:
Thanks for this comment!